sn_node_manager/src/bin/cli/main.rs

```diff
@@ -449,6 +453,26 @@ pub enum DaemonSubCmd {
     Stop {},
 }
 
+#[derive(Subcommand, Debug)]
+pub enum RpcSubCmd {
+    // Issue an RPC command to restart nodes
+    #[clap(name = "restart")]
+    Restart {
+        /// The peer ID of the service to restart.
+        ///
+        /// The argument can be used multiple times to restart many services.
+        #[clap(long)]
+        peer_id: Vec<String>,
+        /// The address of the RPC server where the command will be sent.
+        /// It should be an IP address and port, e.g, 127.0.0.1:59091.
```

Comment:
Sorry, it may seem petty, but, there should be a line break between these lines. It's because of the way it gets rendered on the CLI:
```
Options:
      --peer-id <PEER_ID>
          The peer ID of the service to restart.

          The argument can be used multiple times to restart many services.

      --rpc-server-address <RPC_SERVER_ADDRESS>
          The address of the RPC server where the command will be sent. It should be an IP address and port, e.g, 127.0.0.1:59091

      --retain-peer-id
          Set to retain the peer ID of the nodes to be restarted

  -h, --help
          Print help (see a summary with '-h')
```

Reply:
Yeah was wondering that when the change was suggested but i wasn't sure. I'll send it through